src/components/taskDetails/TaskDetailsSection.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Details from './Details';
+import TaskDropDown from '../tasks/TaskDropDown';
+import ProgressContainer from '../tasks/card/progressContainer';
+import task, { taskStatusUpdateHandleProp } from '@/interfaces/task.type';
+import styles from './task-details.module.scss';
+
+interface TaskDetailsSectionProps {
+    isEditing: boolean;
```

Comment:
again can we make it isEditingMode?

Reply:
if I changed it to `isEditingMode` I have to change it into many files not just here